### DISCUSSION TITLE
Dados do equipamento, estrutura da OS, otimização para celular

### DISCUSSION THREAD

**Discussion by apoionacional** [RamonSilva20/mapos#1856] [General]
Olá a todos, primeiramente parabéns pela ferramenta, tenho a alguns finais de semana melhorado e adaptado para nosso uso aqui, fiz algumas correções e modificações bem específicas para uso nosso que acredito ser útil para outras empresas que trabalham com assistência técnica de equipamentos controlados por número de série, inclui a função de impressão do documento de coleta, reformulação total do zero da estrutura da OS, inclusão de campos de equipamento, envio do arquivo PDF como anexo pelo whatsapp e outros...
Estou terminando a inclusão de um qrcode para verificação única (sem sequencial) da autenticidade da ordem de serviço de maneira online pra evitar fraudes aqui, vinculando um sistema de cadastro mobile de novos clientes ao banco de dados, com confirmação via sms, impressão das etiquetas de entrada para equipamentos e acessórios, barra superior funcional, impressão da etiqueta de saída para postagem já que 80% dos nossos atendimentos é de outros estados...
Vou sair de férias no mês que vem e vou verificar uma forma de ajudar no projeto aqui, também aprender a maneira certa de usar o github, acabei de me cadastrar.
![editar_os](https://user-images.githubusercontent.com/108497625/176794524-52b84e05-c73f-4dd3-978b-4f4026318994.png)
![os](https://user-images.githubusercontent.com/108497625/176794720-ed0aeff1-c7a4-48e8-b0fb-6dc86c9741cf.png)
![barra](https://user-images.githubusercontent.com/108497625/176796240-6b269459-2ba5-4a73-8fcb-f7a929f5d410.png)

**Comment by Edivandro** [comment]
Muito Bom, PARABÊNS!
Ansioso pelas mudanças que virão.

**Reply by Ftech-Games** [reply]
Excelente. estou ansioso pela atualização!

**Comment by brunowachsmann** [comment]
Boa tarde, tudo bem?

Teria como disponibilizar essas mudanças?
